src/test/groovy/org/sharegov/mdcgis/PropertyInfoServiceTests.groovy

```diff
@@ -57,14 +57,9 @@ class PropertyInfoServiceTests {
 		Map propertyInfo = propertyInfoService.getPropertyInfo(857869.749956198, 488913.9375316277)
 		propertyInfo.with{
 			assert parcelFolioNumber == '3059010240130'
-			assert parcelInfoLegal1 == "GLEN COVE WEST SEC 1"
-			assert parcelInfoLegal2 == "PB 118-30"
-			assert parcelInfoLegal3 == "LOT 13 BKL 1"
-			assert parcelInfoLegal4 == "LOT SIZE 8800 SQ FT"
-			assert parcelInfoLegal5 == "OR 17101-5181 0296 1"
-			assert parcelInfoLegal6 == "OR 17101-5181 0296 00"
+			assert parcelInfoLegal == "GLEN COVE WEST SEC 1 PB 118-30 LOT 13 BKL 1 LOT SIZE 8800 SQ FT OR 17101-5181 0296 1"
 			assert parcelInfoPtxAddress == "11826 SW 97 ST"
-			assert parcelInfoAddress == "11826 SW 97TH ST"
+			assert parcelInfoAddress == "11826 SW 97 ST"
```

Comment:
That surprises me, that 97 is not 97TH. What is parcelInfoPtxAddress vs parcelInfoAddress ?